experimental/framework/fluid-static/src/rootDataObject.ts

```diff
@@ -27,6 +28,7 @@ interface RootDataObjectProps {
 
 // eslint-disable-next-line @typescript-eslint/ban-types
 export class RootDataObject extends DataObject<{}, RootDataObjectProps> {
+    private readonly opHandler = (message: ISequencedDocumentMessage) => this.emit("op", message);
```

Comment:
opHandler is intentionally added here and not to FluidContainer as we don't want to expose this on the developer surface